carts/views.py

```diff
@@ -1,3 +1,30 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http import JsonResponse
+from django.views import View
+from django.db.models import Sum, F
+
+from carts.models import Cart
+from users.models import User
+from products.models import Product, Size
+from users.utils import login_deco
+
+class CartView(View):
+    @login_deco
+    def get(self, request):            
+        carts = Cart.objects.filter(user_id = request.user.id)
+        total_price = carts.aggregate(price = Sum(F("count") * F("product__price")))
+
+        response = [{
+            "cart_id" : cart.id,
+            "name"    : cart.product.name,
+            "image"   : cart.product.image_url,
+            "color"   : cart.product.color.name,
+            "size"    : cart.size.name,
+            "count"   : cart.count,
+            "price"   : int(cart.count * cart.product.price)}
+            for cart in carts]
```

Comment:
```suggestion
        response = [{
            "cart_id" : cart.id,
            "name"    : cart.product.name,
            "image"   : cart.product.image_url,
            "color"   : cart.product.color.name,
            "size"    : cart.size.name,
            "count"   : cart.count,
            "price"   : int(cart.count * cart.product.price)
        } for cart in carts]
```
list comprehension 잘 적용하셨습니다. 
구조만 다음과 같이 수정하면 가독성이 더 올라 갈 것 같습니다.